plugins/ufm_log_analyzer_plugin/src/loganalyze/log_analyzers/console_log_analyzer.py

```diff
@@ -106,6 +109,11 @@ def print_exceptions_per_time_count(self):
             "Exceptions count",
         )
 
+    def save_ufm_versions(self):
+        self._txt_for_pdf.append(
+            f"Used ufm version in console log {self.ufm_versions}{os.linesep}"
```

Comment:
With the change you did, I'm not sure we need the os.linesep here. And if we do, it does not make sense to have it here, I would move it to the PDF class